plugins/Basic Animated Theme/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "author": "Warhead",
+    "id": "f42146a4-bb22-4ebc-856b-44c02e0211bd",
+    "version": "1.0.0",
+    "name": "Basic Animated Background-Theme",
+    "link": "cave-enhancements.com/",
+    "description": "This theme uses never before properties to create a really awesome looking basic animated background for your bridge. expirence. For Dark Theme users ONLY. ",
```

Comment:
This descriptions seems to be grammatically incorrect, has a spelling error and an unnecessary space at the end. Please change it to something like...
`"This theme uses properties never used  before to create a really awesome looking animated background to level up your bridge. experience. For Dark Theme users ONLY."`

Afterwards, we should be good to add your plugin 👍